src/DynamoCore/Logging/Analytics.cs

```diff
@@ -19,6 +21,15 @@ internal static void Start(DynamoModel model)
         {
             client = new DynamoAnalyticsClient();
             client.Start(model);
+            model.WorkspaceAdded += OnWorkspaceAdded;
```

Comment:
Analytics is not a static instance right? otherwise it will hold model if not release the event handler. For example, open/close Dynamo multiple times in Revit.